packages/docs/src/components/sidebar/Sidebar.vue

```diff
@@ -128,7 +128,7 @@ export default defineComponent({
       routeHasActiveChild,
       isActiveChildRoute,
       onSidebarItemClick,
-      badgeColors: { wip: 'primary', new: 'danger', updated: 'warning' },
+      badgeColors: { wip: 'primary', new: 'danger', updated: 'warning', beta: 'danger' },
```

Comment:
Mb we should use different colors for new/beta?

Reply:
Need to clarify with @xx13.